.github/workflows/main-ML.yml

```diff
@@ -0,0 +1,37 @@
+name: Docs Publish
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        language: [it_IT, ja_JP, ko_KR, ml_IN, ru_RU, es_UN, ta_IN, tr_TR, vi_VN]
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install Qiskit machine-learning dependencies
+      run: |
+        cd machine-learning
+        python -m pip install --upgrade pip
+        pip install -U virtualenv setuptools wheel
+        pip install jupyter sphinx_rtd_theme 'matplotlib<3.3.0' qiskit-terra[visualization] 'torchvision<0.10.0'
+        sudo apt-get install -y pandoc graphviz
+        pip install -U -r requirements-dev.txt -r requirements.txt sphinx-intl
+        pip install -e .
```

Comment:
Is this needed? Won't this just install the qiskit metapackage, do you need to do that for the machine learning docs?

Reply:
yeah, I agree. I'll add these in the script instead